third-party/eazy/v1/openapi.yaml

```diff
@@ -459,13 +459,20 @@ paths:
           schema:
             type: string
             example: '458e633a-ab50-445f-9d1f-52afbf720ec3'
-        - in: body
-          name: name
-          required: true
-          description: Name of the team to be created
-          schema:
-            type: string
-          example: 'Support'
+      requestBody:
+        description: The team you would like to create
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - name
+              properties:
+                name:
+                  description: Name of the team to be created
+                  type: string
+                  example: Support
```

Comment:
Is this correct? `body` is not a valid `in:` value. The request bodies are not rendered. E.g. https://api.tyntec.com/reference/support_inbox/current.html?python#support-inbox-api-Create%20a%20team .

Reply:
yes, thank you